.circleci/config.yml

```diff
@@ -18,7 +50,42 @@ jobs:
       MAVEN_OPTS: -Xmx3200m
 
     steps:
+      - abort_for_docs
+      - abort_for_noci
+      - early_return_for_forked_pull_requests
```

Comment:
@chayim why do you return so early for forked?

Reply:
Change in latest to the end.